app/assets/javascripts/admin/addon/components/webhook-status.gjs

```diff
@@ -1,12 +1,14 @@
 import Component from "@glimmer/component";
+import { hash } from "@ember/helper";
+import icon from "discourse-common/helpers/d-icon";
 import I18n from "discourse-i18n";
 
 export default class WebhookStatus extends Component {
   iconNames = ["far-circle", "times-circle", "circle", "circle"];
   iconClasses = ["text-muted", "text-danger", "text-successful", "text-muted"];
 
   get status() {
```

Comment:
wdyt about popping a `@cached` on it (since it's called 3 times)

Reply:
I don't think it's worth it